skills/csharp/experimental/itsmskill/Utilities/LocaleTemplateManagerWrapper.cs

```diff
@@ -0,0 +1,156 @@
+
+        private const string CardsOnly = "CardsOnly";
+
+        public static Activity GetCardResponse(this LocaleTemplateManager manager, Card card)
+        {
+            return manager.GetCardResponse(new Card[] { card });
+        }
+
+        public static Activity GetCardResponse(this LocaleTemplateManager manager, IEnumerable<Card> cards, string attachmentLayout = "carousel")
+        {
+            return manager.GetCardResponse(CardsOnly, cards, null, attachmentLayout);
+        }
+
+        public static Activity GetCardResponse(this LocaleTemplateManager manager, string templateId, Card card, IDictionary<string, object> tokens = null)
+        {
+            return manager.GetCardResponse(templateId, new Card[] { card }, tokens);
+        }
+
+        public static Activity GetCardResponse(this LocaleTemplateManager manager, string templateId, IEnumerable<Card> cards, IDictionary<string, object> tokens = null, string attachmentLayout = "carousel")
+        {
+            if (string.IsNullOrEmpty(templateId))
+            {
+                templateId = CardsOnly;
+            }
+
+            var input = new
+            {
+                Data = tokens,
+                Cards = cards.Select((card) => { return Convert(card); }).ToArray(),
+                Layout = attachmentLayout,
+            };
+            try
+            {
+                return manager.GenerateActivityForLocale(templateId, input);
+            }
+            catch (Exception ex)
+            {
+                var result = Activity.CreateMessageActivity();
+                result.Text = ex.Message;
+                return (Activity)result;
+            }
+        }
+
+        public static Activity GetCardResponse(this LocaleTemplateManager manager, string templateId, Card card, IDictionary<string, object> tokens = null, string containerName = null, IEnumerable<Card> containerItems = null)
+        {
+            throw new Exception("1. create *Containee.json which only keeps containee's body;2. in the container, write ${if(Cards==null,'',join(foreach(Cards,Card,CreateStringNoContainer(Card.Name,Card.Data)),','))}");
+
+            if (string.IsNullOrEmpty(templateId))
+            {
+                templateId = CardsOnly;
+            }
+
+            var input = new
+            {
+                Data = tokens,
+                Cards = new CardExt[] { Convert(card, containerItems: containerItems) },
+            };
+            try
+            {
+                return manager.GenerateActivityForLocale(templateId, input);
+            }
+            catch (Exception ex)
+            {
+                var result = Activity.CreateMessageActivity();
+                result.Text = ex.Message;
+                return (Activity)result;
+            }
+        }
+
+        public static Activity GetResponse(this LocaleTemplateManager manager, string templateId, IDictionary<string, object> tokens = null)
```

Comment:
do we want to change GetResponse and GetCardResponse to GenerateActivity?

Reply:
Updated